### PR TITLE
End to End browser test framework and first test

### LINKED ISSUE #89: Implement easy end-to-end testing
To make sure features / hotfixes are didn't break anything I would recommend to implement a simple end-to-end testing system.

For this I would recommend [Cypress](https://www.cypress.io/), which is very straight forward ans easy manageable with npm.

This will also improve the ability to detect bugs faster.

The default test command will run End-To-End tests in a headless electron browser.

As an alternativ tests can also run with chrome or firefox.

I will submit a Pull-Request with the framework and a first test in the next minutes

### PR DESCRIPTION
**Pull request by dannylinden**
PR to resolve #89 

Test can be run as:

- Electron headless browser: `npm test`
- Chrome browser: `npm test:chrome`
- Firefox headless browser: `npm test:firefox`

Chrome and Firefox need to be available on the machine, Electron, the default is shipped with Cypress.

I already added two Testcases as start:

- Validate the phone numbers are displayed and correct.
- Validate the Appstore Badges are visible and correct linked (Itunes & Google)

More Infos:

- [https://www.cypress.io/](https://www.cypress.io/)
- [https://docs.cypress.io/guides/getting-started/writing-your-first-test.html](https://docs.cypress.io/guides/getting-started/writing-your-first-test.html)

### CODE REVIEW

**Comment by dannylinden** [comment]
Here an example run:

Bildschirmfoto 2020-06-17 um 14 17 14

**Comment by dannylinden** [comment]
One info: Node 10 is required, but i didn't add this as a requirement to the package.json yet, because i don't know if this break any build pipeline and its still not needed for the build itself.

**Comment by dannylinden** [comment]
I have added one more test case which validates the app store badges are visible and link to the correct url

**Comment by tkowark** [comment]
Thanks a lot for that. Next case I would add is to check whether the FAQ deeplinks are unique. More will probably emerge over time, but this is a really good start. Can this be merged before the webp PR?

**Comment by dannylinden** [comment]
@tkowark yes, both are independently to merge to master :)

File: README.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ After a successful build, you'll have a new folder ```public``` in the repositor
 ```
 It will automatically use ```public``` as base directory, go to ```localhost:8080``` to view the website.
 
+### Testing
+[Cypress](https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell) is used to run End-To-End tests. tests are located in the ```cypress/integration``` folder and can be run with:
+```bash
+  npm test
```
Comment (tkowark) [inline README.md:49]: Running npm tests seems to create folder cypress/integration, cypress/videso and cypress/support. Shall these be added to the to the .gitignore?

Comment (dannylinden) [reply README.md:49]: cypress/integration containing the test cases, so this definitiv not. The following can be added to gitignore:

- cypress/plugins (currently no plugins needed)
- cypress/screenshots (screenshot from failed tests)
- cypress/support (currently no support code needed)
- cypress/videos (video from failed tests)

I will ad an commit for this

**Pull request merged**